### PR TITLE
feat: tags change from RSP@v3

### PR DESCRIPTION
**Pull request by jianliao**
## Description

1. Rename variant name from `Deletable` to `Removable`
2. Introduced a new modifier class named - `spectrum-Tags-itemIcon`
3. Add a new property `is-selected`
4. Remove `deletable` variant example

## How and where has this been tested?
 - Latest Chrome/Firefox/Safari on MacOS and IE11 and Edge on Windows 10

## Screenshots

image

image

## To-do list

- [x] If my change impacts other components, I have tested to make sure they don't break.
- [x] If my change impacts documentation, I have updated the documentation accordingly.
- [x] I have read the [CONTRIBUTING document](/.github/CONTRIBUTING.md).

- [x] This pull request is ready to merge.

### CODE REVIEW

File: components/tags/index.css [hunk 1]
```diff
@@ -74,3 +76,14 @@ governing permissions and limitations under the License.
   white-space: nowrap;
   text-overflow: ellipsis;
 }
+
+.spectrum-Tags-item--removable {
+
+  .spectrum-Tags-itemClearButton {
+    &.is-focused {
+      border: 2px solid;
```
Comment (bernhard-adobe) [inline components/tags/index.css]: no DNA var for that border? like this one maybe ? http://spectrograph.corp.adobe.com/token-browser/token/36222?q=tag%20removeable

File: components/tags/skin.css [hunk 2]
```diff
@@ -10,62 +10,106 @@ OF ANY KIND, either express or implied. See the License for the specific languag
...
+    &.is-invalid {
+      border-color: var(--spectrum-tag-border-color-error-selected);
+      background-color: var(--spectrum-tag-background-color-error-selected);
+      .spectrum-Tags-itemIcon,
+      .spectrum-Tags-itemLabel,
+      .spectrum-Tags-itemClearButton {
+        color: var(--spectrum-tag-text-color-error-selected);
+      }
+      &.focus-ring {
+        box-shadow: 0 0 0 2px var(--spectrum-tag-border-color-error-selected-key-focus);
```
Comment (bernhard-adobe) [inline components/tags/skin.css]: I think DNA beta 6.0 has that focus ring variable:
https://git.corp.adobe.com/Spectrum/spectrum-dna/tree/6.0-beta

`        "spectrum-alias-focus-ring-size": "2px",`  Not seeing one for tags there yet.

**Comment by GarthDB** [comment]
This doesn't actually seem in sync with design, which means either design needs to be updated, or React did something separate. 

Snap shot from current design xd file:
image

**Comment by GarthDB** [comment]
Looks like the UI kit doesn't have a selected state.

**Comment by jianliao** [comment]
I've confirmed the selected state with Lindsay.  She pointed me to the latest update of tag component. https://www.dropbox.com/s/76ajnbqdbj7usag/tag-update_2.0.0.xd?dl=0

**Comment by misterbrownlee** [comment]
This seems ready to merge, so here goes...

**Pull request merged**